bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -182,6 +184,19 @@ public Integer getSample() {
                                               }
                                           });
         initializeExplicitLacFlushPolicy();
+
+        if (bk.getConf().getAddEntryQuorumTimeout() > 0) {
```

Comment:
can AddEntryQuorumTimeout be disabled ? I think this would be an error. Maybe we can add some assertion and fail fast. We can do it in the future, not blocker

Reply:
@eolivelli how do you mean it would be an error? If add entry quorum timeout is disabled, the timeouts will only be on the individual add entry RPCs sent to bookies.

Reply:
@ivankelly got it, thank you. I had another (wrong) idea in my mind